docs/details/clickhouse.md

```diff
@@ -0,0 +1,41 @@
+# ClickHouse
+
+
+You can use an external Clickhouse database instance outside the PMM Server container running on other hosts.
+
+## Environment variables
+
+PMM predefines certain flags that allow you to use ClickHouse parameters as environment variables:
+
+To use ClickHouse as an external database instance, use the following environment variables: 
+ 
+`PERCONA_TEST_PMM_CLICKHOUSE_ADDR` -> hostname:port
+:   Name of the host and port of the external ClickHouse database instance. 
+
+**Optional environment variables**
+
+`PERCONA_TEST_PMM_CLICKHOUSE_DATABASE` -> database name
+:   Database name of the external ClickHouse database instance.
+
+`​​PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE` -> [pool size](https://github.com/ClickHouse/ClickHouse/blob/master/programs/server/config.xml#L1130)
```

Comment:
is this a right link ? https://github.com/ClickHouse/ClickHouse/blob/master/programs/server/config.xml#L288 probably?

Should we link to the some commit so it wouldn't change? @nikita-b

Reply:
Yes you are right, but I think we'll need to clarify that it's the same setting but for connection: 
```
The maximum number of threads in the current connection thread pool. Can't be bigger then: max_thread_pool_size: https://github.com/ClickHouse/ClickHouse/blob/ba26b3cf4c982f681f7f9782bba2662f97620835/programs/server/config.xml#L288
```